### PR TITLE
deprecating scion-viz

### PR DESCRIPTION
**Pull request by mwfarb**
This PR removes install/test directions for scion-viz and updating for webapp.

---
This change is [Reviewable](https://reviewable.io/reviews/netsec-ethz/scion-tutorials/102)

### CODE REVIEW

File: content/as_visualization/webapp_development.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+cd ${GOPATH}/src/github.com/netsec-ethz/scion-apps/camerapp/imageserver
+go install imageserver.go
+go run ${GOPATH}/src/github.com/netsec-ethz/scion-apps/webapp/tests/imgtest/imgserver/local-image.go &
+imageserver -s 1-ffaa:0:112,[127.0.0.2]:42002 -sciondFromIA &
+```
+Now, from your webapp browser interface running on your virtual client SCION node, you can enter both client and server addresses and ask the client for the most recently generated remote image.
+
+## Related Links
+* [Webapp AS Visualization](../as_visualization/webapp.md)
+* [Webapp SCIONLab Apps Visualization](../as_visualization/webapp_apps.md)
```
Comment (juagargi) [inline content/as_visualization/webapp_development.md:72]: I would remove this one as it links to this page again.

**Review by juagargi — CHANGES_REQUESTED** [review]
Only one comment.
I am going to rebase and push the newly built `html` files on a separate commit.
We typically send two commits in `scion-tutorials`: 1 for the "real" contents (the `.md` files, images, etc), and one separate commit for the built `html` files (what you get after running `mkdocs build`). This eases review and rebasing.

**Review by mwfarb — COMMENTED** [review]
Thanks! I can see how that would help. :-) 

> *[Reviewable](https://reviewable.io/reviews/netsec-ethz/scion-tutorials/102#-:-LgmuBps7LnmmQHANyDv:b917ekw)* status: 0 of 62 files reviewed, 1 unresolved discussion (waiting on @juagargi)

___
*[content/as_visualization/webapp_development.md, line 72 at r1](https://reviewable.io/reviews/netsec-ethz/scion-tutorials/102#-Lg8ukt204RwW9rZEljf-r1-72:-Lgms-w7BSsD9Qio7ftn:bogfwq6) ([raw file](https://github.com/netsec-ethz/scion-tutorials/blob/a28eb2bdeea2632313672a98be95c96da4c3720c/content/as_visualization/webapp_development.md#L72)):*
<details><summary><i>Previously, juagargi (Juan A. García Pardo Giménez de los Galanes) wrote…</i></summary><blockquote>

I would remove this one as it links to this page again.
</blockquote></details>

I'm not seeing a recursive link in the `webapp_development.md` file. I think you meant the recursive link in the `webapp_apps.md` file, I fixed that one.

**Review by juagargi — APPROVED** [review]
:lgtm:

> Reviewed 17 of 62 files at r1, 41 of 44 files at r3, 4 of 4 files at r4.
> *[Reviewable](https://reviewable.io/reviews/netsec-ethz/scion-tutorials/102#-:-Lgn1h0WEbTH6sns2XXY:bnfp4nl)* status: :shipit: complete! all files reviewed, all discussions resolved

**Pull request merged**